setup-local-env.sh

```diff
@@ -14,3 +14,4 @@ export AWS_REGION=us-east-1
 export DYNAMODB_TABLE_NAME=efcms-local
 export USER_LIMITER_THRESHOLD='5000'
 export IP_LIMITER_THRESHOLD='5000'
+export PDF_EXPRESS_LICENSE_KEY='OjkUB41bl1hJg6jvUEfn'
```

Comment:
this looks half real; i wouldn't want to be committing real keys to the codebase.

Reply:
Good catch!  @TomElliottFlexion, I know you were working on the PDF viewer stuff.  Is this a real license key that must be kept secret?  Please ping someone else if I got the wrong person.

Reply:
Removed!